module-presentation/src/main/java/com/depromeet/followingLog/dto/response/FollowingLogMemoryResponse.java

```diff
@@ -0,0 +1,194 @@
+
+    @Schema(
+            description = "수영 기록 일기",
+            example = "오늘 수영을 열심히 했다",
+            requiredMode = Schema.RequiredMode.NOT_REQUIRED)
+    private String diary;
+
+    @Schema(
+            description = "총 수영 거리",
+            example = "175",
+            requiredMode = Schema.RequiredMode.NOT_REQUIRED)
+    private Integer totalDistance;
+
+    @Schema(description = "달성 여부", example = "false", requiredMode = Schema.RequiredMode.REQUIRED)
+    private boolean isAchieved;
+
+    @Schema(
+            description = "소모한 칼로리",
+            example = "100",
+            requiredMode = Schema.RequiredMode.NOT_REQUIRED)
+    private Integer kcal;
+
+    @Schema(
+            description = "영법 타입(NORMAL, SINGLE, MULTI)",
+            example = "NORMAL",
+            requiredMode = Schema.RequiredMode.REQUIRED)
+    private String type;
+
+    @Schema(description = "영법별 거리 리스트", requiredMode = Schema.RequiredMode.NOT_REQUIRED)
+    private List<StrokeResponse> strokes;
+
+    @Schema(description = "이미지", requiredMode = Schema.RequiredMode.NOT_REQUIRED)
+    private String imageUrl;
+
+    @Schema(
+            description = "팔로잉 소식 생성 시간",
+            example = "2024-08-16 00:00:00",
+            requiredMode = Schema.RequiredMode.REQUIRED)
+    private String createdAt;
+
+    @Schema(
+            description = "팔로잉 소식 최신 유무",
+            example = "false",
+            requiredMode = Schema.RequiredMode.REQUIRED)
+    private boolean isRecentNews;
+
+    @Builder
+    public FollowingLogMemoryResponse(
+            Long memberId,
+            String memberNickname,
+            Long memoryId,
+            String recordAt,
+            String startTime,
+            String endTime,
+            Short lane,
+            String diary,
+            Integer totalDistance,
+            boolean isAchieved,
+            Integer kcal,
+            String type,
+            List<StrokeResponse> strokes,
+            String imageUrl,
+            String createdAt,
+            boolean isRecentNews) {
+        this.memberId = memberId;
+        this.memberNickname = memberNickname;
+        this.memoryId = memoryId;
+        this.recordAt = recordAt;
+        this.startTime = startTime;
+        this.endTime = endTime;
+        this.lane = lane;
+        this.diary = diary;
+        this.totalDistance = totalDistance;
+        this.isAchieved = isAchieved;
+        this.kcal = kcal;
+        this.type = type;
+        this.strokes = strokes;
+        this.imageUrl = imageUrl;
+        this.createdAt = createdAt;
+        this.isRecentNews = isRecentNews;
+    }
+
+    public static FollowingLogMemoryResponse toFollowingLogMemoryResponse(
+            FollowingMemoryLog followingMemoryLog, LocalDateTime lastViewedFollowingLogAt) {
+        String createdAt =
+                followingMemoryLog
+                        .getCreatedAt()
+                        .format(DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss"));
```

Comment:
```suggestion
        LocalDateTime createdAt = followingMemoryLog.getCreatedAt();
```
제가 알기로 yyyy-MM-dd HH:mm:ss 패턴의 문자열이 LocalDateTime으로 변환이 잘 되는 걸로 알아서,
이렇게 하면 어떨까 제안 드립니다.
`@Schema(example = "2024-08-16 00:00:00")`만 지정해주면 Swagger에서 보기 편하게 보일 수 있습니다!

Reply:
시도 결과 그냥 LocalDateTime으로 반환 시 `yyyy-MM-ddTHH:mm:ss:SSS` 형태로 변환이 되는 것을 확인하였습니다. 현재는 `@JsonFormat` 를 활용하여 `yyyy-MM-dd HH:mm:ss` 로 변환시켰습니다.

- `@JsonFormat 적용 전`
![createdAt LocalDateTime](https://github.com/user-attachments/assets/72b3e019-82c5-4c27-8012-585e3c9672ec)

- `@JsonFormat 적용 후`
![createdAt jsonformat 적용 후](https://github.com/user-attachments/assets/93230483-12e6-41e9-ada0-870378a84c2f)

Reply:
이거 프론트엔드와도 상의가 된 내용인가요??

Reply:
상의 후 알려드리겠습니다.

Reply:
상의 결과 그렇게 보내줘도 된다고 하셨습니다